test/DebugInfo/debug_compilation_dir_rel.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t
+// RUN: cd %t
+// RUN: cp %s .
+// RUN: %target-swift-frontend -g \
+// RUN:   -wmo -c -debug-compilation-dir path/to \
+// RUN:   debug_compilation_dir_rel.swift -o - -emit-ir | %FileCheck %s
+
+func foo() {}
+
+// CHECK: !DIFile(filename: "debug_compilation_dir_rel.swift", directory: "path/to")
```

Comment:
Test looks good, I would put both in the same test file though (and use the driver in both cases). You can use a different check prefix for each case and run FileCheck with `--check-prefix=`, eg:

```
// RUN: %target-swiftc_driver -g \
// RUN:   -wmo -c -debug-compilation-dir /path/to \
// RUN:   %s -o - -emit-ir | %FileCheck --check-prefix=CHECK-ABS %s
// RUN: %empty-directory(%t)
// RUN: mkdir -p %t
// RUN: cd %t
// RUN: cp %s .
// RUN: %target-swiftc_driver -g \
// RUN:   -wmo -c -debug-compilation-dir /path/to \
// RUN:   debug_compilation_dir_rel.swift -o - -emit-ir | %FileCheck --check-prefix=CHECK-REL %s

func foo() {}

// CHECK-ABS: !DIFile(filename: "{{.*}}/debug_compilation_dir.swift", directory: "/path/to")
// CHECK-REL: !DIFile(filename: "debug_compilation_dir_rel.swift", directory: "/path/to")
```

Reply:
Thanks for the advice!

Btw, should I test relative path case with absolute compilation directory path `/path/to` or relative one `path/to`?

Reply:
I think the relative / absolute of the compilation directory doesn't matter so much, we can leave it as absolute for the test